operations/CSM_product_management/Perform_NCN_Personalization.md

```diff
@@ -16,7 +16,7 @@ to create a [configuration layer](../configuration_management/Configuration_Laye
 Products may supply multiple plays to run, in which case multiple configuration
 layers must be created. Consult the manual for each product to configure them on
 NCNs by referring to the [1.5 HPE Cray EX System Software Getting Started Guide
```

Comment:
Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
NCNs by referring to the [HPE Cray EX System Software Getting Started Guide
```